Dockerfile.functional

```diff
@@ -14,6 +14,7 @@ COPY test ./test
 COPY Makefile ./
 COPY manifests/ ./manifests
 COPY products ./products
+COPY version ./version
```

Comment:
@MStokluska we had to manually add the version directory in order for the test images to compile.
Is there some reason why we don't add the whole directory and rather add file/directory by file/directory? 🤔